website/docs/learn-graphql/guides/thinking-in-graphql.md

```diff
@@ -0,0 +1,147 @@
+- that multiple back-end services might return different subsets for what is, semantically speaking, the same conversation object
+- or even the very act of fetching the data from the various back-end services
+
+These are the types of implementation details that you want to abstract away from complex data-driven UI code, as that makes it easier to reason about and thus to maintain/optimise over time. Additionally, when thinking of complex applications, you will want to encapsulate business logic in a way that allows you to re-use it for other UIs, or perhaps even compute/transform the data in a background thread.
+
+All of these needs are met by a GraphQL schema that acts as the abstraction layer between your back-end and front-end.
+
+## The “graph” in GraphQL
+
+Another important UI consideration is rendering performance, or sometimes perceived performance. The former is achieved by having all data available that is necessary, for the initial state of the UI that the user should see, such that only a single render pass is required. (Sometimes this might mean that it can take a little while longer before rendering can start, but even then a single render pass can still provide an improvement to perceived performance.)
+
+Ideally all this data can be provided within a reasonable time-frame, but even then there are provisions in state-of-the-art GraphQL stacks that allow you to design a controlled loading experience using [the “render-as-you-fetch” pattern](https://17.reactjs.org/docs/concurrent-mode-suspense.html#traditional-approaches-vs-suspense), as outlined in [this in-depth presentation](https://www.youtube.com/watch?v=Tl0S7QkxFE4) by a Facebook/Relay engineer.
+
+All in all, what this means is that the schema _should_ enable a piece of UI to fetch all data it needs, in a single request. This is where “the graph” comes in, which means that the types that make up the schema are connected to each other in semantically meaningful ways and can be retrieved as a meaningful whole.
+
+:::note
+
+### Broad-Query
+
+This concept might seem foreign even to those already familiar with GraphQL. To solve this at Microsoft, we had to go as far as invent a new name for this very core concept: **Broad-Query**.
+
+However, because in GraphQL _all_ queries are meant to be “broad”, we will **not** keep repeating the “Broad-Query” term. After all, we want you to walk away from this guide as someone who truly understands GraphQL!
+:::
+
+### 👍 Design from front-end perspective
+
+When designing the schema in a vacuum, it might be hard to imagine what those connections should be. However, when considered from the perspective of a concrete piece of UI, and working your way backwards, it actually becomes a lot easier.
+
+Let's consider the conversation list UI example again:
+
+<table>
+<tr>
+<th>Conversation list</th>
+<th>UI components</th>
+<th>GraphQL query</th>
+</tr>
+<tr>
+<td>
+<img src={require("./images/SmallChatList.png").default} border="1" />
```

Comment:
Image broken in preview here, not sure if there is an issue once it is in the 'real' view

Reply:
Works 👍 https://microsoft.github.io/graphitation/docs/learn-graphql/guides/thinking-in-graphql#-design-from-front-end-perspective